NetTopologySuite.Samples.Console/Tests/Various/GraphBuilder.cs

```diff
@@ -1,137 +0,0 @@
-using System;

```

Comment:
We can do cleanup of the code later, let's not try to achieve absolute perfection on this pullrequest alone... ;-)